modules/cats-effect/README.md

```diff
@@ -0,0 +1,51 @@
+# Cats-effect module for PureConfig
```

Comment:
Not mandatory but we highly recommended to use tut to generate the markdown files. See what we do for the [cats module](https://github.com/pureconfig/pureconfig/tree/master/modules/cats/src/main/tut).

Reply:
Ok, will do so